doc/authentication.md

```diff
@@ -75,6 +76,13 @@ To allow establishing a new trust relationship from the client side, you must se
 In a production setup, unset `core.trust_password` after all clients have been added.
 This prevents brute-force attacks trying to guess the password.
 
+(authentication-token)=
+#### Adding client certificates using tokens
+
+New clients can be added using tokens. This is a safer way than using the trust password, as tokens expiry once they've been used. For this, the server needs to generate a token and pass it to the client.
```

Comment:
Updated suggestion:
```suggestion
You can also add new clients by using tokens. This is a safer way than using the trust password, because tokens expire once they've been used. 

To use this method, generate a token for each client by calling `lxc config trust add`, which will prompt for the client name.
The clients can then add their certificates to the server's trust store by providing the generated token when prompted for the trust password. 
Alternatively, the clients can provide the token directly when adding the remote: `lxc remote add <name> <token>`
```